### PR TITLE
Deploy registration-operator using manifests

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: deploy/managedcluster/hub/install.sh [hunk 1]
```diff
@@ -22,12 +22,25 @@ cd registration-operator || {
...
+if [ "${ns}" -eq 0 ]; then
+  $KUBECTL create ns open-cluster-management
 fi
 
+$KUBECTL apply -f deploy/cluster-manager/crds/0000_01_operator.open-cluster-management.io_clustermanagers.crd.yaml
+$KUBECTL apply -f deploy/cluster-manager/cluster_role.yaml
+$KUBECTL apply -f deploy/cluster-manager/cluster_role_binding.yaml
+$KUBECTL apply -f deploy/cluster-manager/service_account.yaml
+$KUBECTL apply -f deploy/cluster-manager/operator.yaml
+$KUBECTL apply -f deploy/cluster-manager/crds/
```
Comment (qiujian16) [inline deploy/managedcluster/hub/install.sh:41]: rm this?

File: test/e2e/util/util.go [hunk 2]
```diff
@@ -378,10 +378,10 @@ func CreateManagedCluster(dynamicClient dynamic.Interface) (*unstructured.Unstru
 	}
 
 	// update fakeManagedCluster status with the new client
-	err = SetStatusType(fakeManagedCluster, clusterv1.ManagedClusterConditionJoined)
-	if err != nil {
-		return nil, err
-	}
+	// err = SetStatusType(fakeManagedCluster, clusterv1.ManagedClusterConditionJoined)
```
Comment (qiujian16) [inline test/e2e/util/util.go:381]: is this intentional?

File: deploy/managedcluster/hub/install.sh (hunk 1 above)
Comment (zhiweiyin318) [reply deploy/managedcluster/hub/install.sh]: done

File: test/e2e/util/util.go (hunk 2 above)
Comment (zhiweiyin318) [reply test/e2e/util/util.go:381]: yes. the status could not impact the e2e.
 it reports error  even if I update the lastTransitionTime with time.Now().Format("9090-09-09T51:951:08Z").
` ManagedCluster.cluster.open-cluster-management.io "test-automation-bxtgw" is invalid: status.conditions.lastTransitionTime: Invalid value: "9090-09-09T51:951:08Z": status.conditions.lastTransitionTime in body must be of type date-time: "9090-09-09T51:951:08Z"`

Comment (qiujian16) [reply test/e2e/util/util.go:381]: so why we comment them instead removing them?

Comment (zhiweiyin318) [reply test/e2e/util/util.go:381]: I think the error could be fixed by the upgraded api when the condition PR is merged.

Comment (qiujian16) [reply test/e2e/util/util.go:381]: gotcha

**Pull request merged**